server.js

```diff
@@ -2,24 +2,25 @@ var express = require('express');
 var app = express();
 var fs = require('fs');
 var path = require('path');
+var bodyParser = require('body-parser');
 
 app.use(express.static(path.join(__dirname, '/public')));
-app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Technically, I don't think this change was needed for these simple data types.  Did this correct an issue?